src/sw.test.js

```diff
@@ -212,8 +212,12 @@ describe('Service Worker', () => {
       'https://static.files.bbci.co.uk/ws/simorgh-assets/public/igbo/images/icons/icon-72x72.png?v=1',
       'https://static.test.files.bbci.co.uk/ws/simorgh-assets/public/igbo/images/icons/icon-144x144.png?v=2',
       'https://static.files.bbci.co.uk/ws/simorgh-assets/public/igbo/images/icons/icon-144x144.png?v=2',
-      // Reverb
-      'https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/reverb-3.9.2.js',
+      // Reverb - preview1, preview2, test & live
```

Comment:
We also need to update https://github.com/bbc/simorgh/blob/latest/cypress/e2e/specialFeatures/serviceWorker/assertions/index.ts#L63-L64